packages/bootleg-common/bin/bootleg-coverage-ignore

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const mm = require('micromatch');
+
+const coverageFile = path.join(process.cwd(), 'coverage/coverage.json');
+const coverageData = require(coverageFile);
+
+const ignorePatterns = ['**/node_modules/**', '**/Migrations.sol'];
+const coveredFiles = Object.keys(coverageData);
+
+if (coveredFiles.length < 1) {
+  console.error("Couldn't find any covered files");
+  process.exit(1);
+}
+
+const ignoredCoverageData = coveredFiles
```

Comment:
sol-coverage just instruments _everything_. I'ma go open an issue on their side for a supported way to do this.

This script just ignores everything in node_modules and `migrations.sol`.

Reply:
👍